src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/ApplicationTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing;
+using Xunit.Abstractions;
+
+namespace System.Windows.Forms.UITests;
+
+public class ApplicationTests : ControlTestBase
+{
+    public ApplicationTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    [WinFormsFact]
+    public async Task Application_OpenForms_RecreateHandle()
+    {
+        await RunTestAsync(form =>
+        {
+            form.Show();
+            using Form child = new() { RightToLeft = RightToLeft.No };
+            child.Show(form);
+
+            int formCnt = 2;
```

Comment:
Looks like missed it first time around.
```suggestion
            const int expectedFormCount = 2;
```

Reply:
Completely forgot about such feature 😔